### DISCUSSION TITLE
When will the Card component become stable?

### DISCUSSION THREAD

**Discussion by yiqing-zhao** [microsoft/fluentui#27088] [Q&A] [answered]
As titled. 

Our product used the `Card` component from `@fluentui/react-northstar` and would like to migrate to use Fluent UI v9. However, the Card component is still unstable. The roadmap listed the target date for the `Card` component to become stable as `Early CY23Q1`.

I would appreciate if anyone could give a more detailed and confirmed release schedule for this component.

**Comment by spmonahan** [comment, marked as answer]
`Card` is stable starting in [9.18.0](https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-components/CHANGELOG.md#9180).
